### DISCUSSION TITLE
PrimeFaces 13.0.0 uses Coloris!

### DISCUSSION THREAD

**Discussion by melloware** [mdbassit/Coloris#118] [Show and tell]
Just wanted to thank @mdbassit for this killer component. PF 13.0.0 is now using it as its color picker for JSF. 

See: https://www.primefaces.org/showcase/ui/input/colorPicker.xhtml

**Comment by mdbassit** [comment]
That is excellent news! I'm happy my work was useful to you.
I would also like to thank you for your help getting Coloris on npm.

**Comment by jepsar** [comment]


**Reply by melloware** [reply]
Yep and it includes the caveat about only one per page.

**Comment by mdbassit** [comment]
I'll think about a way to make popover and inline modes coexist without growing the code too much.
